sources/operator.1.18.0.psmdb-operator.json

```diff
@@ -0,0 +1,115 @@
+            "critical": false
+          },
+          "6.0.18-15": {
+            "image_path": "percona/percona-server-mongodb:6.0.18-15",
+            "image_hash": "d197ce16ab0eed6df25e632b92dea5ce448e549e02028f39b78f5730c2ffef36",
+            "status": "recommended",
+            "critical": false
+          },
+          "6.0.16-13": {
+            "image_path": "percona/percona-server-mongodb:6.0.16-13",
+            "image_hash": "1497e58e39497d8425ccd053898dc323338d6eb3f0e3c4c223f9d5a468da7931",
+            "status": "available",
+            "critical": false
+          },
+          "6.0.15-12": {
+            "image_path": "percona/percona-server-mongodb:6.0.15-12",
+            "image_hash": "f12dd271d78cf3e70088fea0c420e8c03703457d8a5959b645053546bff94dea",
+            "status": "available",
+            "critical": false
+          },
+          "6.0.9-7": {
+            "image_path": "percona/percona-server-mongodb:6.0.9-7",
+            "image_hash": "5ef8404e200a680a67f0a94599963e17c029ebe5e0045b60b45062bba127c505",
+            "status": "available",
+            "critical": false
+          },
+          "6.0.5-4": {
+            "image_path": "percona/percona-server-mongodb:6.0.5-4",
+            "image_hash": "b6f875974c59d8ea0174675c85f41668460233784cbf2cbe7ce5eca212ac5f6a",
+            "status": "available",
+            "critical": false
+          },
+          "5.0.29-25": {
+            "image_path": "percona/percona-server-mongodb:5.0.29-25",
+            "image_hash": "732eb60f2d3750aaf654ada7f52d2869928be885f6b7012666d6777f85411b74",
+            "status": "recommended",
+            "critical": false
+          },
+          "5.0.28-24": {
+            "image_path": "percona/percona-server-mongodb:5.0.28-24",
+            "image_hash": "7f9ac418bcd22fc8fbcf6ddba9aff3142a07ddfdfbe58efd5d55d5f7c9f43aaf",
+            "status": "available",
+            "critical": false
+          },
+          "5.0.26-22": {
+            "image_path": "percona/percona-server-mongodb:5.0.26-22",
+            "image_hash": "50873aa99f36319c5590ad2bddf407b4df44728bee86025ccae1bfed9329a0d1",
+            "status": "available",
+            "critical": false
+          },
+          "5.0.20-17": {
+            "image_path": "percona/percona-server-mongodb:5.0.20-17",
+            "image_hash": "56a19362b1082c37eb5e7069d05f7bb281a09c4788101faeea15a50bb8a49e8b",
+            "status": "available",
+            "critical": false
+          },
+          "5.0.15-13": {
+            "image_path": "percona/percona-server-mongodb:5.0.15-13",
+            "image_hash": "f0b5a8291d778d7419c20dcf0d1985a0f33770d05e94dba41db8f071957e9929",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "pmm": {
+          "2.43.2": {
+            "image_path": "percona/pmm-client:2.43.2",
+            "image_hash": "c927c9ff7113132aaf153799bc5cded3ad924040d16cb010a891a76c03deeecf",
```

Comment:
I think this is not the correct hash, it must be the amd64 one, not the multiplatform

Reply:
done